packages/icons/src/components/EthereumTokenIcon.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import { Canvas, ImageSVG, useSVG } from '@shopify/react-native-skia';
+
+import { EthereumTokenIconName, ethereumTokenIcons } from '../icons';
+import { CryptoIconSize, cryptoIconSizes } from './CryptoIcon';
+
+type CryptoIconProps = {
+    name?: EthereumTokenIconName;
+    size?: CryptoIconSize;
+};
+
+export const EthereumTokenIcon = ({ name = 'erc20', size = 'small' }: CryptoIconProps) => {
```

Comment:
Couldn't this reuse the CryptoIcon component? The content is just the same. Then the export of `{ CryptoIconSize, cryptoIconSizes } from './CryptoIcon';` would not be needed and the logic for rendering crypto and token icon would be unified.

Reply:
Yes, it would be possible. We discussed this with @Nodonisko and decided to go this way instead and try to separate it from crypto icon. Altough there is some repetition, which is also there between normal icon and crypto icon, we will now be able to scale this better. And if we ever decide to remove this duplicity, it's very easy refactor.